auto_submit/lib/validations/ci_successful.dart

```diff
@@ -222,7 +222,19 @@ class CiSuccessful extends Validation {
     return dateTime.compareTo(DateTime.now().subtract(const Duration(hours: Config.kGitHubCheckStaleThreshold))) < 0;
   }
 
-  bool isEngineRoller(Author author, github.RepositorySlug slug) {
+  /// Perform stale check only on Engine related rolled PRs.
+  ///
+  /// This includes those rolled PRs from upstream to Engine repo and those
+  /// rolled PRs from Engine to Framework.
+  bool shouldStale(Author author, github.RepositorySlug slug) {
```

Comment:
Should this be something like isStale rather than shouldStale?

Reply:
or markStale?

Reply:
Updated to `supportStale`.